Configurateur/front/lib/screens/container-creation/design_screen.dart

```diff
@@ -94,7 +94,7 @@ class DesignScreenState extends State<DesignScreen> {
 
   @override
   void initState() {
-    checkToken();
+    //checkToken();
```

Comment:
Why keep a commented line ?

Reply:
I need it for testing, it's will be changed with the PR of design screen UI